test/valid-pipelines/trigger.yml

```diff
@@ -76,3 +76,9 @@ steps:
 
   - trigger: "a-slug"
     allow_dependency_failure: true
+
+  - trigger: "a-slug"
+    skip: true
+
+  - trigger: "a-slug"
+    skip: "reason for skipping"
```

Comment:
Probably unimportant, but there should be a newline at the end of the file.
I'm never sure if it's worth mentioning, but GitHub considers it important enough to visually flag it I guess.

Reply:
Fixed 👍